### ISSUE TITLE
Shift+Up/Down

### ISSUE THREAD

**Issue by Magnus26** [musescore/MuseScore#24864]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Open Edit -> Preferences -> Shortcuts.
2. In the list to the right, look up Add to selection: staff below.
3. Try to change that operation's shortcut to Shift+Down. Here on my computer this is not possible but gives just Down instead.

### Supporting files, videos and screenshots

![Screenshot_20240922_081549](https://github.com/user-attachments/assets/b9a84428-fa04-4637-9eb8-7eda7d7c6e97)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2

### Regression

I was unable to check

### Operating system

Q4OS

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by Jojo-Schmitz** [comment]
As far as I can tell left and right arrow have the same issue
As do Ins, Home, Del, End, PageUp and Pagedown

**Comment by Jojo-Schmitz** [comment]
All function keys (F1 - F12 at least, some keybords may have more) are affected too

**Comment by Magnus26** [comment]
I find it strange that Shift+Up actually works for selecting all the notes in a chord on a staff from the bottom note and up. Shift+Down does not make the corresponding selection from the top note and down. I would, however, want the shortcuts to extend the selection one note at a time and not comprise the whole chord immediately.
